### DISCUSSION TITLE
"{{username}}" doesnt have access to "*.{{alias}}.vercel.app"

### DISCUSSION THREAD

**Discussion by sanjaybora04** [vercel/community#7217] [Help] [answered]
### Summary

i was trying to add `"www.{{alias}}.vercel.app"` to my project, i already have `"{{alias}}.vercel.app"` in it but it gives this error
`"{{username}}"` doesnt have access to `"*.{{alias}}.vercel.app"`, i also have `"{{alias}}.tech"` in that project.

i have another project which allows me to add `www.{{alias}}.vercel.app` because i have added `"{{alias}}.com"` in that project

How do i get access to `"www.{{alias}}.vercel.app"` without buying a `{{alias}}.com` domain?

### Steps to Reproduce

* Add a `"{{alias}}.vercel.app"` to your project
* Try to add a `"www.{{alias}}.vercel.app"` to that project. ( You shouldnt have `"{{alias}}.com"` added in that project )

**Comment by swarnava** [comment, marked as answer]
Hi,

- You can add `www.domain.com` if you have access to `domain.com`
- you can only add `subdomain.vercel.app` in your project. You cannot add `www` or any `other.subdomain.vercel.app` in your project.

Also subdomain typically doesn't require `www`.

**Reply by sanjaybora04** [reply]
i am able to add `other.{{alias}}.vercel.app` if i own `{{alias}}.com`  in one of my other projects

[a comment by an automated account is not shown]
